atom/index.test.ts

```diff
@@ -424,3 +424,37 @@ test('can use previous value in subscribers', () => {
   unbind()
   clock.runAll()
 })
+
+test('notifies the subscribed listener with current and old values for a store that had an initial value', () => {
+  let events: (number | undefined)[] = []
+  let $store = atom<number>(1)
+
+  $store.subscribe((value, oldValue?: number) => {
+    events.push(oldValue, value)
+  })
+
+  deepStrictEqual(events, [undefined, 1])
```

Comment:
No, the `value` here should be `1` since store is already initialized with `1` on the moment when we call `subscribe`